source/code/include/scxsystemlib/memoryinstance.h

```diff
@@ -166,7 +166,7 @@ namespace SCXSystemLib
         bool m_foundTotalPhysMem;                       //< Was "MemTotal:" line found in /proc/meminfo
         bool m_foundAvailMem;                           //< Was "MemFree:" line found in /proc/meminfo
         bool m_foundTotalSwap;                          //< Was "SwapTotal:" line found in /proc/meminfo
-        bool m_foundAvailSwap;                          //< Was "SwapFree:" line found in /proc/meminfo
+        bool m_foundAvailSwap;                          //< Was "SwapFree:" line found in /proc/meminfo   
```

Comment:
Looks like you inadvertantly added some spaces here. Remove so the file isn't updated as part of this commit.